### ISSUE TITLE
Cachalot tries to create a database connection on import

### ISSUE THREAD

**Issue by hansegucker** [noripyt/django-cachalot#241]
## What happened?

Directly on import of `cachalot.utils`, `django-cachalot` tries to create a database connection (changed in 2.6.0).

This happens in line 39 in `cachalot.utils`.

Example traceback from a Django app:

```
Traceback (most recent call last):
  File "/path/to/env/bin/aleksis-admin", line 5, in <module>
    from aleksis.core.__main__ import aleksis_cmd
  File "/path/to/env/lib/python3.11/site-packages/aleksis/core/__init__.py", line 3, in <module>
    from .celery import app as celery_app  # noqa
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/to/env/lib/python3.11/site-packages/aleksis/core/celery.py", line 10, in <module>
    from .util.core_helpers import get_site_preferences
  File "/path/to/env/lib/python3.11/site-packages/aleksis/core/util/core_helpers.py", line 22, in <module>
    from cachalot.api import invalidate
  File "/path/to/env/lib/python3.11/site-packages/cachalot/api.py", line 12, in <module>
    from .utils import _invalidate_tables
  File "/path/to/env/lib/python3.11/site-packages/cachalot/utils.py", line 39, in <module>
    connection.ensure_connection()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/to/env/lib/python3.11/site-packages/django/utils/connection.py", line 15, in __getattr__
    return getattr(self._connections[self._alias], item)
                   ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/path/to/env/lib/python3.11/site-packages/django/utils/connection.py", line 60, in __getitem__
    if alias not in self.settings:
                    ^^^^^^^^^^^^^
  File "/path/to/env/lib/python3.11/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ^^^^^^^^^^^^^^^^^^^
  File "/path/to/env/lib/python3.11/site-packages/django/utils/connection.py", line 45, in settings
    self._settings = self.configure_settings(self._settings)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/to/env/lib/python3.11/site-packages/django/db/utils.py", line 148, in configure_settings
    databases = super().configure_settings(databases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/to/env/lib/python3.11/site-packages/django/utils/connection.py", line 50, in configure_settings
    settings = getattr(django_settings, self.settings_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/to/env/lib/python3.11/site-packages/django/conf/__init__.py", line 102, in __getattr__
    self._setup(name)
  File "/path/to/env/lib/python3.11/site-packages/django/conf/__init__.py", line 82, in _setup
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Requested setting DATABASES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
```

## What should've happened instead?

The database connection should be only created while using functions in `cachalot.utils`, not on import. An import alone never should create any database connections.

## Steps to reproduce

Import `cachalot.utils` before Django settings are ready.

**Comment by Andrew-Chen-Wang** [comment]
Which version of cachalot are you using?

**Comment by Andrew-Chen-Wang** [comment]
Dang, sorry about this unfortunate bug. My fault for not double checking. Please let me know if #242 fixes this issue.

**Comment by hansegucker** [comment]
> Dang, sorry about this unfortunate bug. My fault for not double checking. Please let me know if #242 fixes this issue.

Yeah, the fix seems to work. Thank you for your fast reaction.

**Comment by Andrew-Chen-Wang** [comment]
published hot fix 2.6.1

**Comment by hansegucker** [comment]
Thank you very much!
